Sources/TestsCommon/CommonTestUtils.swift

```diff
@@ -132,7 +132,7 @@ open class MongoSwiftTestCase: XCTestCase {
     }
 
     public static var multipleMongosLoadBalancedURI: String? {
-        ProcessInfo.processInfo.environment["MULTIPLE_MONGOS_LB_URI"]
+        ProcessInfo.processInfo.environment["MULTI_MONGOS_LB_URI"]
```

Comment:
this is actually what the evergreen setup tools name the env variable, I had it wrong